geo-test-fixtures/src/lib.rs

```diff
@@ -101,6 +101,24 @@ where
     line_string("shell.wkt")
 }
 
+// From https://geodata.nationaalgeoregister.nl/kadastralekaart/wfs/v4_0?request=GetFeature&service=WFS&srsName=EPSG:4326&typeName=kadastralekaartv4:perceel&version=2.0.0&outputFormat=json&bbox=165593,480993,166125,481552
+pub fn nl_zones<T>() -> MultiPolygon<T>
```

Comment:
These are the polygons from the example in https://github.com/georust/geo/issues/822